.travis.yml

```diff
@@ -7,7 +7,7 @@ python:
 env:
  - DJANGO_VERSION="Django>=1.8,<1.9"
  - DJANGO_VERSION="Django>=1.9,<1.10"
- - DJANGO_VERSION="Django==1.10b1"
+ - DJANGO_VERSION="Django>=1.10"
```

Comment:
I'd use `>=1.10,<1.11` to prevent this from pulling 1.11+ when they are later released :-)